lms/services/h_api.py

```diff
@@ -94,9 +94,7 @@ def _api_request(self, method, path, body=None, headers=None):
                 headers=headers,
                 **request_args,
             )
-        except HTTPError as err:
-            response = getattr(err, "response", None)
```

Comment:
`ExternalRequestError` always has a `response` attr (which may be `None`) so it's not necessary to use `getattr()` here